WordPress/Classes/ViewRelated/Reader/ReaderReblogAction.swift

```diff
@@ -1,5 +1,9 @@
 /// Encapsulates a command to reblog a post
 class ReaderReblogAction {
+    // tells if the origin is the reader list or detail, for analytics purposes
+    enum OriginType {
```

Comment:
This name is a little vague. I wonder if something like `ReblogOrigin` or `ReblogSource` would be more obvious? Especially when it's actually used. For example, without context, this `originType` is a bit confusing.

`action.execute(readerPost: readerPost!, origin: controller, originType: .list)`

Reply:
Yes, ReblogSource has more context for sure